src/components/Card.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import '../styles/card.scss';
+
+export const Card = ({ card, onClick, isFlipped, inactive }) => {
+    const { type, image } = card;
+
+    return (
+        <div className={inactive ? 'cardContainer disappear' : 'cardContainer'} >
+            <div
+                className={isFlipped ? 'card is-flipped' : 'card'}
+                onClick={onClick}
+            >
+                <div className="card__face card__face--front" aria-label={type}></div>
```

Comment:
Nice css naming pattern. However remember you can self close the html tag if empty content.
```suggestion
                <div className="card__face card__face--front" aria-label={type} />
```